src/components/Dashboard/TodoList.vue

```diff
@@ -603,18 +627,29 @@
             <PaymentPending v-else />
           </template>
 
+          <!-- is this an authorization approved continuation item? -->
+          <template v-else-if="EnumUtilities.isStatusApproved(item)">
+            <NameRequestInfo :nameRequest="item.nameRequest" />
+          </template>
+
           <!-- is this a change-requested item? -->
           <template v-else-if="EnumUtilities.isStatusChangeRequested(item)">
             <div class="todo-list-detail body-2">
+              <p
+                v-if="getNameRequest"
+                class="list-item__subtitle"
+              >
+                NR APPROVED - {{ expiresText(getNameRequest) }}
+              </p>
               <p class="list-item__subtitle">
-                This {{ item.title }} is paid but requires you to make the following changes:
+                BC Registries staff require the following changes to your authorization documentation
```

Comment:
Ask Jacqueline if there should be a `:` at the end of the line.

Reply:
checking now